### PR TITLE
[NFDIV-3155] Update money property page and applicant steps

### PR DESCRIPTION
**Pull request by adamg-hmcts**
### Change description ###

The money property page text and applicant steps have been adjusted to match the new financial order [user journey in Miro](https://miro.com/app/board/uXjVM43UCCA=/).

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-3155

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/app/case/to-api-format.ts [hunk 1]
```diff
@@ -316,13 +316,13 @@ const fields: ToApiConverters = {
   coApplicant2StatementOfTruth: data => ({
     coApplicant2StatementOfTruth: checkboxConverter(data.coApplicant2StatementOfTruth),
   }),
-  applicant1WhoIsFinancialOrderFor: data => ({
-    applicant1FinancialOrdersFor:
-      data.applicant1ApplyForFinancialOrder === YesOrNo.YES ? data.applicant1WhoIsFinancialOrderFor : [],
+  applicant1ApplyForFinancialOrder: data => ({
```
Comment (DelythJustice) [inline src/main/app/case/to-api-format.ts:319]: Could you have a comment on this explaining the change maybe?

File: src/main/steps/applicant1/dividing-money-property/content.ts [hunk 2]
```diff
@@ -1,46 +1,94 @@
...
-  disagreeOnDividingAssets: 'Os ydych yn anghytuno ynghylch rhannu arian ac eiddo',
-  disagreeOnDividingAssetsDetails:
-    "Gallwch ofyn i'r llys benderfynu ar eich rhan. Gelwir hyn yn gofyn i'r llys wneud 'gorchymyn ariannol'. Mae hyn yn golygu y bydd y llys yn penderfynu sut y caiff asedau eu rhannu. Gallwch hefyd wneud cais am orchymyn ariannol i'ch plant, os yw'n briodol. Gall y llys hefyd orchymyn i daliadau cynhaliaeth gael eu gwneud.",
-  readMore: 'Darllenwch fwy am gynhaliaeth plant',
-  readMoreContent: `Dim ond o dan rai amgylchiadau y gall y llys wneud gorchmynion ariannol i blant. Gallwch ddod i gytundeb â'ch ${partner} a'i wneud yn gyfreithiol rwymol gyda gorchymyn cydsynio, neu gallwch ddefnyddio'r Gwasanaeth Cynhaliaeth Plant.`,
-  costs: `Rhaid i chi wneud cais ar wahân i'r llys am orchymyn ariannol, gan ddefnyddio ffurflen arall. Mae'n costio ${getFee(
-    config.get('fees.financialOrder')
-  )} yn ychwanegol.`,
+const cy: typeof en = ({ partner, isDivorce }: CommonContent) => ({
```
Comment (DelythJustice) [inline src/main/steps/applicant1/dividing-money-property/content.ts:40]: Do we need the welsh translation on this before it goes in?

Comment (BWeeks-JusticeGovUK) [reply src/main/steps/applicant1/dividing-money-property/content.ts:40]: @DelythJustice Welsh translations are usually handled in their own ticket.

File: src/main/app/case/to-api-format.ts (hunk 1 above)
Comment (adamg-hmcts) [reply src/main/app/case/to-api-format.ts:319]: In the previous user journey, we had 1 form that contained radio buttons to select whether they wanted a financial order (`applicant1ApplyForFinancialOrder`) and checkboxes to select who it should apply to (`applicant1WhoIsFinancialOrderFor`). They submitted the form which contained data for both fields and if they had selected No for `data.applicant1ApplyForFinancialOrder`, we discarded any checkbox selections for `applicant1WhoIsFinancialOrderFor` by it that to a blank array.

In the new user journey, `applicant1ApplyForFinancialOrder` and `applicant1WhoIsFinancialOrderFor` are answered on different pages in different forms. We never receive the form data with both answers, but if at any point they answer No to `applicant1ApplyForFinancialOrder`, we add `applicant1FinancialOrdersFor: []` to the body of the post request to ensure we blank out that field in their case data. For example, they could begin their application initially by selection Yes for `applicant1ApplyForFinancialOrder` and choosing an option for `applicant1WhoIsFinancialOrderFor`. Then if they then go back and change `applicant1ApplyForFinancialOrder` to No, we want to ensure that `applicant1WhoIsFinancialOrderFor` is not still set in the database.

File: src/main/steps/applicant1/dividing-money-property/content.ts (hunk 2 above)
Comment (adamg-hmcts) [reply src/main/steps/applicant1/dividing-money-property/content.ts:40]: Yeah we had a chat about the Welsh in one of our team meetings and Kate seemed to imply we could merge with just the English to start with. I’ll highlight that the Welsh is missing again with a comment in the ticket when it goes to awaiting sign-off

File: src/test/steps/happy-path.ts [hunk 3]
```diff
@@ -177,9 +177,6 @@ Given("I've completed all happy path questions correctly", async () => {
   iClick('Continue');
 
   I.waitInUrl('/dividing-money-property');
-  iClick('Continue');
```
Comment (DelythJustice) [inline src/test/steps/happy-path.ts:180]: happy path still needs the pages tho right? and a no

**Review by DelythJustice — DISMISSED** [review]

File: src/test/steps/happy-path.ts (hunk 3 above)
Comment (adamg-hmcts) [reply src/test/steps/happy-path.ts:180]: We've moved the form from `/do-you-want-to-apply-financial-order` to `/dividing-money-property` and dropped the `/do-you-want-to-apply-financial-order` page from the user journey. As that financial order page doesn't exist anymore, the happy path now is to click `No` on `/dividing-money-property` and then continue straight on to `/upload-your-documents`

**Review by DelythJustice — APPROVED** [review]
Just a coment that there is still no welsh on this one

**Pull request closed without merging**